### PR TITLE
Fix classnames

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | A few classnames are incorrect.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | /
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11668)

### CODE REVIEW

File: install-dev/index_cli.php [hunk 1]
```diff
@@ -37,7 +37,7 @@
 $_SERVER['REQUEST_URI'] = '/install/index_cli.php';
 require_once dirname(__FILE__).'/init.php';
 require_once(__DIR__).DIRECTORY_SEPARATOR.'autoload.php';
-require_once _PS_INSTALL_PATH_.'classes/datas.php';
+require_once _PS_INSTALL_PATH_ . 'classes/Datas.php';
```
Comment (mickaelandrieu) [inline install-dev/index_cli.php]: How about `session.php` and `exception.php`?

The names of the files are inconsistent with the classes inside

**Review by mickaelandrieu — CHANGES_REQUESTED** [review]
2 more fixes to be done, then LGTM

File: install-dev/index_cli.php (hunk 1 above)
Comment (MathiasReker) [reply install-dev/index_cli.php]: Sure :)

Comment (PierreRambaud) [reply install-dev/index_cli.php]: @mickaelandrieu It's a BC if you rename files :/

Comment (MathiasReker) [reply install-dev/index_cli.php]: @PierreRambaud Agree, but does anyone override install files?

Comment (PierreRambaud) [reply install-dev/index_cli.php]: We don't know, but we can't add BC in 1.7.6.

**Comment by MathiasReker** [comment]
Fixed BC

**Comment by MathiasReker** [comment]
Wdyt?

File: src/PrestaShopBundle/Install/Simplexml.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
 use DOMDocument;
 
-class SimplexmlElement extends \SimpleXMLElement
+class Simplexml extends \SimpleXMLElement
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Install/Simplexml.php:31]: Hum, changing this class can break something :/ More if this classe is not loaded with the autoloader.

**Comment by MathiasReker** [comment]
@PierreRambaud Should I better close this PR again, to be safe?

**Comment by PierreRambaud** [comment]
@MathiasReker I think :/ Really sorry

**Comment by MathiasReker** [comment]
No problem :)

**Pull request closed without merging**